src/ui/components/ServerLaunchSection.css

```diff
@@ -101,8 +104,24 @@
   box-shadow: 0 0 5px 3px rgba(0, 0, 0, 0.2);
 }
 
-.ServerConfig .TextInputWrapper {
-  display: inline-block;
+.ServerConfig {
```

Comment:
In all honesty, when combined with reverting back to the `backdrop-blur`, the server launch section should go back to both checkboxes on the same line. It's more compact.

Also as a general UI/UX design rule, controls (such as checkboxes) should always be aligned while the text should be the part that changes width/position, rather than the controls moving with the length of the text.